ingest/rules/fetch_from_ncbi.smk

```diff
@@ -124,3 +124,23 @@ rule format_ncbi_datasets_ndjson:
             --duplicate-reporting warn \
             2> {log} > {output.ndjson}
         """
+
+###########################################################################
+########################## 2. Fetch from Entrez ###########################
+###########################################################################
+
+rule fetch_from_ncbi_entrez:
+    params:
+        term=config["entrez_search_term"],
```

Comment:
_non-blocking_

Just had the thought that we could construct this search term in the workflow using the `ncbi_taxon_id` so that we don't have to worry about the taxid getting out of sync in the config.

```suggestion
        term=f'txid{config["ncbi_taxon_id"]}[Primary Organism]',
```